micropython/senml/examples/base.py

```diff
@@ -0,0 +1,35 @@
+# _  __  ____    _   _
+# | |/ / |  _ \  | \ | |
+# | ' /  | |_) | |  \| |
+# | . \  |  __/  | |\  |
+# |_|\_\ |_|     |_| \_|
+#
+# (c) 2018 KPN
+# License: MIT license.
+# Author: Jan Bogaerts
+#
+# base value example
+
+from senml import *
+import utime as time
+
+
+pack = SenmlPack("device_name")
+temp = SenmlRecord(
+    SenmlNames.KPN_SENML_TEMPERATURE, unit=SenmlUnits.SENML_UNIT_DEGREES_CELSIUS, value=23.5
```

Comment:
This `KPN_SENML_TEMPERATURE` constant can be replaced simply with the string `"temperature"`.  Then there's no need for the special KPN names (which don't really have any official meaning, as far as I know).